Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -373,24 +353,18 @@ open class ChartDataSet: ChartBaseDataSet
     /// - returns: The array-index of the specified entry
     ///
     /// - parameter e: the entry to search for
+    // TODO: Should be returning `nil` to follow Swift convention
     open override func entryIndex(entry e: ChartDataEntry) -> Int
     {
-        for i in 0 ..< values.count
-        {
-            if values[i] === e
-            {
-                return i
-            }
-        }
-        
-        return -1
+        return values.index { $0 === e } ?? -1
     }
     
     /// Adds an Entry to the DataSet dynamically.
     /// Entries are added to the end of the list.
     /// This will also recalculate the current minimum and maximum values of the DataSet and the value-sum.
     /// - parameter e: the entry to add
     /// - returns: True
+    // TODO: This should return `Void` to follow Swift convention
```

Comment:
why return Void instead of nothing or Bool?

Reply:
Returning nothing is returning `Void`. This follows the convention in the core Swift i.e. `Array.append`